src/views/Home.jsx

```diff
@@ -14,6 +25,23 @@ export function Home({ makeNewList, handleError }) {
 			</p>
 			{handleError ? <span>{handleError}</span> : null}
 			<button onClick={handleClick}>Create a new list!</button>
+
+			<div className="JoinListForm">
+				<p>Join an existing shopping list by entering a three word token.</p>
+				{joinListErrorMsg ? <span>{joinListErrorMsg}</span> : null}
+				<form onSubmit={handleJoinList}>
+					<label htmlFor="input">Share token</label>
```

Comment:
Utilized the same pattern for forms as @fakehouseplant and @drakenguyen4000 in issue #4 with the onSubmit action on the form instead of on the button.

Reply:
I like all the error handling you guys have included! One (small, aesthetic) suggestion is to move the {handleError ? <span>{handleError}</span> : null} down to below the form and button components, partially for consistency with the error that pops up if you type extra spaces and partially just because it took me a second to realize what had happened when the error popped up in the middle of the text and the form